src/ValidatorInterface.php

```diff
@@ -7,7 +7,7 @@
 /**
  * Validator validates {@link DataSetInterface} against rules set for data set attributes.
  *
- * @psalm-type RulesType = null|RuleInterface|RulesProviderInterface|callable|iterable<RuleInterface|RuleInterface[]|callable|callable[]>
```

Comment:
These types can be mentioned in PHPDoc now. But I can handle it later during adjusting docs.

Reply:
Yes. Here need list types with descriptions.